http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -1596,6 +1698,61 @@ class GKEInterpreter[F[_]](
     ) ++ configs ++ galaxyRestoreSettings
   }
 
+  private[util] def buildRStudioAppChartOverrideValuesString(
+    appName: AppName,
+    cluster: KubernetesCluster,
+    nodepoolName: NodepoolName,
+    namespaceName: NamespaceName,
+    disk: PersistentDisk,
+    ksaName: ServiceAccountName,
+    userEmail: WorkbenchEmail,
+    stagingBucket: GcsBucketName
+  ): List[String] = {
+    val rstudioIngressPath = s"/proxy/google/v1/apps/${cluster.cloudContext.asString}/${appName.value}/rstudio-service"
+    val welderIngressPath = s"/proxy/google/v1/apps/${cluster.cloudContext.asString}/${appName.value}/welder-service"
+    val k8sProxyHost = kubernetesProxyHost(cluster, config.proxyConfig.proxyDomain).address
+    val leoProxyhost = config.proxyConfig.getProxyServerHostName
+
+    val rewriteTarget = "$2"
+    val ingress = List(
+      raw"""ingress.enabled=true""",
+      raw"""ingress.annotations.nginx\.ingress\.kubernetes\.io/auth-tls-secret=${namespaceName.value}/ca-secret""",
+      raw"""ingress.annotations.nginx\.ingress\.kubernetes\.io/proxy-redirect-from=https://${k8sProxyHost}""",
+      raw"""ingress.annotations.nginx\.ingress\.kubernetes\.io/proxy-redirect-to=${leoProxyhost}""",
+      raw"""ingress.annotations.nginx\.ingress\.kubernetes\.io/rewrite-target=/${rewriteTarget}""",
+      raw"""ingress.hosts[0].host=${k8sProxyHost}""",
+      raw"""ingress.hosts[0].paths[0].path=${rstudioIngressPath}${"(/|$)(.*)"}""",
+      raw"""ingress.hosts[0].paths[0].backend.service.name=aou-rstudio""",
+      raw"""ingress.hosts[0].paths[0].backend.service.port.number=8787""",
+      raw"""ingress.hosts[0].paths[1].path=${welderIngressPath}${"(/|$)(.*)"}""",
+      raw"""ingress.hosts[0].paths[1].backend.service.name=aou-rstudio-welder""",
+      raw"""ingress.hosts[0].paths[1].backend.service.port.number=8080""",
+      raw"""ingress.tls[0].secretName=tls-secret""",
+      raw"""ingress.tls[0].hosts[0]=${k8sProxyHost}"""
+    )
+
+    val welder = List(
+      raw"""welder.extraEnv[0].name=GOOGLE_PROJECT""",
+      raw"""welder.extraEnv[0].value=${cluster.cloudContext.asString}""",
+      raw"""welder.extraEnv[1].name=STAGING_BUCKET""",
+      raw"""welder.extraEnv[1].value=${stagingBucket.value}""",
+      raw"""welder.extraEnv[2].name=CLUSTER_NAME""",
+      raw"""welder.extraEnv[2].value=${cluster.clusterName.value}""",
```

Comment:
Perhaps this should be the appName?

Reply:
Ohhh, I thought this was the k8s name! But from Welder's perspective this was intended as the cluster as in dataproc cluster name?